packages/docs/src/components/content/MaturityChecklist/MaturityChecklist.tsx

```diff
@@ -1,93 +1,232 @@
 import React from 'react';
 import MaturityChecklistItem, { CheckStatus } from './MaturityChecklistItem';
+import { Accordion, SvgIcon } from '@cmsgov/design-system';
+import AccordionItem from '../AccordionItem';
 
 interface MaturityChecklistProps {
-  // Accessibility
-  a11yStandards: CheckStatus;
-  color: CheckStatus;
-  forcedColors: CheckStatus;
-  screenReaders: CheckStatus;
-  keyboardNavigable: CheckStatus;
-
-  // Code
-  storybook: CheckStatus;
-  responsive: CheckStatus;
-  spanish: CheckStatus;
-
-  // Design
-  completeUiKit: CheckStatus;
-  responsiveUiKit: CheckStatus;
-
-  // Tokens
-  tokensInCode: CheckStatus;
-  tokensInSketch: CheckStatus;
+  children: string;
+
+  items: {
```

Comment:
I don't see a reason why this needed to change. Was this leftover from a previous version of the code? It looks like this could be reverted without any negative effects and down below it the references can be reverted from things like `props.items['forcedColors']` to `props.forcedColors`.

Reply:
Then you'd be able to get rid of all the changes in the diff that just had to do with reformatting the `<MaturityChecklist>` instances